scripts/post-release-notes-preview.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/local/bin/python3
+
+import os
+import requests
+import subprocess
+import sys
+
+BRANCH_PREFIXES_TO_POST = ['feat', 'fix', 'chore', 'test', 'docs', 'revert']
+GIT_BRANCH = os.environ['CIRCLE_BRANCH']
+gitBranchParts = GIT_BRANCH.split('/')
+if len(gitBranchParts) <= 1:
+    sys.exit(0)
+
+gitBranchPrefix = gitBranchParts[0]
+if not gitBranchPrefix in BRANCH_PREFIXES_TO_POST:
+    sys.exit(0)
+
+SLACK_WEBHOOK = os.environ['SLACK_WEBHOOK']
+USERNAME = os.environ.get('CIRCLE_USERNAME', 'USER_NOT_FOUND')
+
+# last version:
+lastVersion = subprocess.check_output(['git', 'describe', '--abbrev=0', '--tags', 'origin/staging'], text=True).strip()
+
+# commits since last version
+missingCommitsHeaders = subprocess.check_output(['git', 'log', '--no-decorate', lastVersion + '..HEAD', '--oneline'], text=True).strip()
+
+# TODO: handle reverts?
+features = []
+fixes = []
+others = []
+
+lines = missingCommitsHeaders.split('\n')
+for line in lines:
+    words = line.split(' ')
+    hash = words[0]
+    prefix = words[1]
+    rest = ' '.join(words[2:])
```

Comment:
What do we expect the rest to look like? Perhaps a comment with some sample content could help clarify here!

Reply:
literally everything in the line output of ```git log --oneline --no-decorate``` after the hash and prefix:
so it's really just the commit message title :)

Reply:
there's no expectations around it - there's just collecting it and displaying it to us